fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepaymentScheduleInstallment.java

```diff
@@ -1706,4 +1706,8 @@ public boolean hasPenalties() {
     public boolean isFullyGraced() {
         return installmentNumber == 0;
     }
+
+    public BigDecimal originalInterestChargedAmount() {
```

Comment:
will this be a column in the table ? try to see where we get the value from

Reply:
Yes. This is a column already added to this table. This column is used to hold the actual interest charged to the installment. Because in case if a repayment is made between the accrual dates, which pays the installments in full, installment interest is recalculated for the days accrued and actual interest amount is stored in this column for later use